README.md

```diff
@@ -24,7 +24,10 @@ source ./venv/bin/activate
 pip3 install -r requirements.txt 
 ```
 
-- Please download Firebase Admin SDK service json file from Firebase Console and put that in project's root directory. Rename file to firebase_cred.json
+- Please download Firebase Admin SDK service json file from Firebase Console, You can download the file by following below steps:
+    1. Go to `Project Settings/Service accounts`
+    1. Click on `Generate New Primary Key`
+    1. Put file in project root directory and rename it to `google-credentials.json`
```

Comment:
`Put file in project root` can be replaced with `Place the file in project's root `.